app/views/shared/bulk_updates/display/_science_keywords.html.erb

```diff
@@ -0,0 +1,21 @@
+<% if bulk_update['update-type'] == 'FIND_AND_REMOVE' || bulk_update['update-type'] == 'FIND_AND_REPLACE' %>
+  <!-- Find Value -->
+  <%= display_value_title(bulk_update['update-type'], 'find') %>
+
+  <ul class="arrow-tag-group-list">
+    <% display_science_keyword(bulk_update['find-value']).each do |field, value| %>
+      <li itemprop="keyword" class="arrow-tag-group-item"><%= value %></li>
+    <% end %>
+  </ul>
+<% end %>
+
+<% if bulk_update['update-type'] == 'ADD_TO_EXISTING' || bulk_update['update-type'] == 'CLEAR_ALL_AND_REPLACE' %>
```

Comment:
I'm going to approve this, but I think this line needs to be changed. I would suggest `<% unless bulk_update['update_type'] == 'FIND_AND_REMOVE' %>`.
The other item was just a thought, I'll leave it to you if you feel like changing anything there.